crossbeam-utils/src/atomic/mod.rs

```diff
@@ -24,9 +24,14 @@ cfg_if::cfg_if! {
 }
 
 #[cfg(not(crossbeam_no_atomic_cas))]
+// We cannot provide AtomicCell under cfg(crossbeam_loom) because loom's atomic
+// types have a different in-memory representation than the underlying type.
+// TODO: The latest loom supports fences, so fallback using seqlock may be available.
+#[cfg(not(crossbeam_loom))]
```

Comment:
(Well, this may be difficult because seqlock has [another issue](http://www.hpl.hp.com/techreports/2012/HPL-2012-68.pdf).)